bedrock/settings/static_media.py

```diff
@@ -984,6 +997,21 @@
         ),
         'output_filename': 'js/contribute-studentambassadors-join-bundle.js',
     },
+    'contribute-taskview': {
+        'source_filenames': (
+            'js/base/global.js',
```

Comment:
Drive by comment - no need to include files again that are already in the main site JS bundle (both `global.js` and `mozilla-client.js`).